src/core/public/doc_links/doc_links_service.ts

```diff
@@ -412,6 +423,82 @@ export class DocLinksService {
           addData: `${OPENSEARCH_WEBSITE_DOCS}`,
           vega: `${OPENSEARCH_DASHBOARDS_VERSIONED_DOCS}`,
           dateMath: `${OPENSEARCH_WEBSITE_DOCS}`,
+          log4j: `https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html#logging`,
+          mappingTypes: `https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html`,
+          scrollSearch: `https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-scroll.html#scroll-search-context`,
+          appendProcessor: `https://www.elastic.co/guide/en/elasticsearch/reference/master/append-processor.html`,
+          bytesProcessor: `https://www.elastic.co/guide/en/elasticsearch/reference/master/bytes-processor.html`,
+          ingestCircleProcessor: `https://www.elastic.co/guide/en/elasticsearch/reference/master/ingest-circle-processor.html`,
+          csvProcessor: `https://www.elastic.co/guide/en/elasticsearch/reference/master/csv-processor.html`,
+          convertProcessor: `https://www.elastic.co/guide/en/elasticsearch/reference/master/convert-processor.html`,
+          dataProcessor: `https://www.elastic.co/guide/en/elasticsearch/reference/master/date-processor.html`,
```

Comment:
Most the links listed here are just for reference purpose. They are hardcoded somewhere in the code base. We may either delete them or fix them in the future. For know, i think it might be fine to just keep it here. Once we have the proper replacement, we should keep it organized like we did in `opensearch` or `opensearchDashboards`.

Reply:
Sure, that's reasonable.